testing/tests/DevExpress.ui.widgets.editors/numberBoxParts/common.tests.js

```diff
@@ -5,7 +5,7 @@ import devices from 'core/devices';
 import eventsEngine from 'events/core/events_engine';
 import keyboardMock from '../../../helpers/keyboardMock.js';
 import pointerMock from '../../../helpers/pointerMock.js';
-import { appendShadowRoot } from '../../../helpers/shadowDOM.js';
+import { get } from '../../../helpers/shadowDOM.js';
```

Comment:
As i understood, now we need to refactor all our tests using `get` and `ensureShadowRootIfNeeded` to be able to run tests in ShadowDom (and start writing tests only using this api). Can we design a solution which help us not to change current testing approach?

As an option, we can add a specific `QUnit.testStart` in `qunitExtension.js` when `Use ShadowSom` checkbox is enabled. For example, this way:
```javascript
            const initjQuery = jQuery.fn.init;
            $.fn.init = function(s, c, r) {
                const result = new initjQuery(s, c, r);
                return new initjQuery(result?.get(0).shadowRoot || result, c, r);
            };
``` 
to get shadowRoot instead of usual dom element + similar approach for shadowDom attach

Reply:
I'll think about this approach

Reply:
done